invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/errors/FormFeedback.js

```diff
@@ -217,7 +217,7 @@ class DisconnectedFormFeedback extends Component {
     const { icon, type } = feedbackConfig[feedback] || {};
 
     return (
-      <Message visible {...{ [type]: true }} className="flashed top attached">
+      <Message visible {...{ [type]: true }} className="flashed top attached" id={type + "-feedback-div"}>
```

Comment:
please check what type can be and add a guard to make sure that it doesn't break at runtime